README.md

```diff
@@ -40,6 +40,12 @@ Then, you can run:
 brew install homebrew-ffmpeg/ffmpeg/ffmpeg --with-<option1> --with-<option2> ...
 ```
 
+If you **really** wish to install all the available options, you can run:
+
+```
+brew install homebrew-ffmpeg/ffmpeg/ffmpeg $(brew options homebrew-ffmpeg/ffmpeg/ffmpeg --compact)
+```
+
```

Comment:
Maybe add:

> Note that this may install libraries for which you need extra SDKs installed, such as `--with-decklink`.

Reply:
Hmm… Also `chromaprint` needs to to be installed using the method explained in the troubleshooting. I will rephrase with there two remarks.